tests/unit/src/flixel/animation/FlxAnimationControllerTest.hx

```diff
@@ -88,6 +88,15 @@ class FlxAnimationControllerTest extends FlxTest
 		Assert.areEqual(1, timesCalled);
 	}
 	
+	@Test // #1786
+	function testNullFrameName():Void
+	{
+		loadSpriteSheet();
+		sprite.animation.addByPrefix("Test", "test");
+		
+		Assert.isNull(sprite.animation.getByName("Test"));
```

Comment:
Hmm, this is actually pretty good still. Makes sure if `findByPrefix` doesn't find anything, no animation gets made. I don't know how necessary of a check that is, though.